### DISCUSSION TITLE
Cameras on IoT network + http -> https

### DISCUSSION THREAD

**Discussion by skidank** [fwestenberg/reolink_dev#544] [Q&A]
I'm running my cameras on a separate Iot network from my hass setup. Also, my hass setup is using https both internally and externally. My plan is to add a rule to allow my cameras selective access to my main network to post webhooks back and I also plan on using socat to convert http requests to https. The one thing I'm not sure about it what the webhook callback URL looks like so that I can appropriately do the mapping. I'm assuming it is generating it from the internal address specified in hass.  If that is the case, it would be really helpful if there was a way to explicitly set the host:port that would be used by the webhook so I can have more control of the situation.

So question is, does the above make sense and will it work?

**Comment by rainbowbreeze** [comment]
I also have an IoT separate network for all my reolink cameras. I set a rule on the firewall to allow all traffic from IoT cameras to the Home Assistant IP on the main network, with not filter of particular ports.

Right now, I don't even know it it' working. Any hint on how to check it?

**Reply by skidank** [reply]
For motion detection, you'd be able to see if the sensor state changes. Personally, I gave up and wrote a small python script to poll my cameras motion state and POST to the hass `/api/states` REST API to update the motion sensors. Not a great solution as it requires constant polling, but it's better than nothing.
